packages/origin-ui-core/src/components/AdminUsersTable.tsx

```diff
@@ -0,0 +1,111 @@
+import { IUser, IOrganization } from '@energyweb/origin-backend-core';
+import { Edit } from '@material-ui/icons';
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { getOffChainDataSource } from '../features/general/selectors';
+import { getUserOffchain } from '../features/users/selectors';
+import {
+    IPaginatedLoaderHooksFetchDataParameters,
+    usePaginatedLoader
+} from './Table/PaginatedLoaderHooks';
+import { TableMaterial } from './Table/TableMaterial';
+import { useHistory } from 'react-router-dom';
+
+interface IRecord {
+    user: IUser;
+}
+
+export const Status = {
+    1: 'Pending',
+    2: 'Active',
+    3: 'Suspended',
+    4: 'Deleted'
+};
+
+export const KYCStatus = {
```

Comment:
Duplicate declaration. This enum can be imported from the `origin-backend-core` package.

Reply:
This are based on object key-value

Reply:
i rename to KYCStatus